service/user/src/test/kotlin/me/helloc/techwikiplus/user/infrastructure/logging/UseCaseLoggingIntegrationTest.kt

```
@@ -0,0 +1,74 @@
+
+    @Test
+    fun `UseCase 실행 시 AOP를 통해 로깅이 동작한다`() {
+        // given
+        val email = "test@example.com"
+        val nickname = "testuser"
+        val password = "Password123!"
+
+        // when - 이메일 전송 실패로 예외가 발생할 것으로 예상
+        try {
+            userSignUpUseCaseWrapper.signUp(email, nickname, password)
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)